src/Unread.ts

```diff
@@ -66,7 +66,7 @@ export function doesRoomHaveUnreadMessages(room: Room): boolean {
     // despite the name of the method :((
     const readUpToId = room.getEventReadUpTo(myUserId!);
 
-    if (!SettingsStore.getValue("feature_thread")) {
+    if (!SettingsStore.getValue("feature_threadstable")) {
```

Comment:
please also update labs/betas.md in element-web :)

Reply:
Not as part of this PR as develop.element.io is broken

Reply:
That file is in a different repo so can't be part of this PR anyway

Reply:
yup, just mentioning it as a step to complete